api/src/main/java/uk/gov/hmcts/payment/api/configuration/security/SpringSecurityConfiguration.java

```diff
@@ -103,6 +103,7 @@ protected void configure(HttpSecurity http) throws Exception {
                 .antMatchers(HttpMethod.GET, "/pba-accounts/*/payments").hasAnyAuthority(PAYMENTS_ROLE,"pui-finance-manager","caseworker-cmc-solicitor", "caseworker-publiclaw-solicitor", "caseworker-probate-solicitor", "caseworker-financialremedy-solicitor", "caseworker-divorce-solicitor")
                 .antMatchers(HttpMethod.GET, "/card-payments/*/status").hasAnyAuthority(PAYMENTS_ROLE, CITIZEN_ROLE)
                 .antMatchers(HttpMethod.GET, "/reference-data/**").permitAll()
+                .antMatchers(HttpMethod.GET, "/case-payment-orders**").permitAll()
```

Comment:
1. API should be whitelisted to Users with Payments & Citizen Role
2. /case-payment-orders** -----are we looking for a change in API Path ? or just '/case-payment-orders' is fine for the current scope?

Reply:
1. @YasharRahvar said to open the endpoint and not limit it to Users with Payments & Citizen Role
2. this is the name we use in CPO microservice. If you need a different name feel free to suggest something.

Reply:
I think just having '/case-payment-orders' would be enough

Reply:
Role permitAll() is fine for now, we might change later if required.